### PR TITLE
Add poke methods to %near-handler in Urbit object

### LINKED ISSUE #1: Implement "urbit" object in Near Social VM
Fork the Near Social VM and add a custom object (in line with their Social, State, Near, and Clipboard objects) called Urbit and wire it up to the http-api library.

### PR DESCRIPTION
**Pull request by bonbud-macryg**
Closes #1 and #3.

My only changes in this PR are in `near/src/lib/vm/vm.js`.

Not ready to merge until #2 is closed.

### CODE REVIEW

File: near/src/lib/vm/vm.js [hunk 1]
```diff
@@ -1853,26 +1853,33 @@ export default class VM {
         },
       },
       Urbit: {
```
Comment (bonbud-macryg) [inline near/src/lib/vm/vm.js:1855]: Why is the `Urbit` object defined inside the return statement with `clipboard` and not outside of it in the `VM` class like `Social` and `Near`?

Comment (hanfel-dovned) [reply near/src/lib/vm/vm.js:1855]: No reason, I just put it in-line at first when I was messing with it. Probably makes more sense to define it with the other larger objects yeah.

**Comment by hanfel-dovned** [comment]
`vm.js` looks good! Any reason to not merge this into the `near-handler` branch right now?

**Comment by bonbud-macryg** [comment]
I was thinking of merging into `master` but merging into `near-handler` might make more sense in this case; it's the quickest way to get something testable. Let me shift `Urbit` up one level and we could merge.

**Pull request merged**